Sources/NIOPosix/System.swift

```diff
@@ -718,25 +714,12 @@ internal enum Posix: Sendable {
         }
     }
 
-    #if !os(Windows)
     @inline(never)
     public static func writev(descriptor: CInt, iovecs: UnsafeBufferPointer<IOVector>) throws -> IOResult<Int> {
         try syscall(blocking: true) {
             sysWritev(descriptor, iovecs.baseAddress!, CInt(iovecs.count))
         }
     }
-    #endif
```

Comment:
This is a `#if !os(Windows)` within a larger `#if !os(Windows)`. Therefore remove.